### PR TITLE
added try-catch to add action

### PR DESCRIPTION
**Pull request by CharlieMantium**
Totally not sure if this is ok. But I tried and it worked.

### CODE REVIEW

File: src/store/activities/actions.js [hunk 1]
```diff
@@ -7,17 +7,21 @@ export const addActivity = activity => ({
...
-    .push(activity)
-    .then(ref => {
-      dispatch(
-        addActivity({
-          id: ref.key,
-          ...activity,
-        }),
-      );
-    });
+  try {
```
Comment (Lukasz-Semik) [inline src/store/activities/actions.js]: `try catch` block is proper, except that you are doing nothing for user :P Think about it in short future, how to prompt users about errors in nice way :) 
I wanted you to use `async/await` for practice, but you still uses `Promises` :) So, I would leave `changes request` 😱

File: src/store/activities/actions.js [hunk 2]
```diff
@@ -6,18 +6,18 @@ export const addActivity = activity => ({
...
-      dispatch(
-        addActivity({
-          id: ref.key,
-          ...activity,
-        }),
-      );
-    });
+export const startAddActivity = (activity = {}) => async dispatch => {
+  try {
+    const id = await database.ref('activities/items').push(activity).key;
```
Comment (Lukasz-Semik) [inline src/store/activities/actions.js]: you can make it more clean:
```
const { key: id } = async....
```

File: src/store/activities/actions.js [hunk 3]
```diff
@@ -6,18 +6,18 @@ export const addActivity = activity => ({
...
+    return dispatch(
+      addActivity({
+        id,
+        ...activity,
+      }),
+    );
+  } catch (error) {
+    return console.error(`Ops! ${error}`); // eslint-disable-line no-console
```
Comment (Lukasz-Semik) [inline src/store/activities/actions.js]: console.logs are returning `undefined`, so there is not need for this return, as there are no code after it.

Comment (CharlieMantium) [reply src/store/activities/actions.js]: eslint didn't allow to remove the return word :/ "9:67  error  Expected to return a value at the end of async arrow function  consistent-return"

**Pull request merged**